README.md

```diff
@@ -73,112 +72,113 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Removed extra spaces when recognizing Chinese, Japanase or Korean languages. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
+- Fixed utility exit logic to close all threads and avoid crashes. (This was a hotfix for 0.62)
+- Fixed an issue where a selection would start on right-click.
 
 ### Installer
 
-- Fixed a regression that was causing the PowerToys shortcut to be deleted on update. (This was a hotfix for 0.61)
-- Updated the .NET dependency to 6.0.8.
+- Added logic to exit PowerToys on upgrade before trying to update .NET.
+- Updated the .NET dependency to 6.0.9.
+- Added clearer installation step names for the boostrapper.
+- Windows App SDK, VC++ redistributable and PowerToys Interop runtime files are now shared between utilites through hardlinks, reducing installation size.
 
 ### Documentation
 
-- Fixed wrong links to installers in README. Thanks [@unuing](https://github.com/unuing)!
+- Fixed typos in Keyboard Manager documentation. Thanks [@eltociear](https://github.com/eltociear)!
+- Replaced docs.microsoft.com links with learn.microsoft.com. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
 
 ### Development
 
-- Removed FXCop leftovers. Thanks [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper)!
-- Added version number to missing binaries and added a CI script to verify that all binaries have their version numbers set correctly.
-- Updated a dependency to fix building on Visual Studio 17.3 C++ tools.
-- Fixed and reactivated the CI unit tests for FancyZones.
-- Cleaned up and removed dead code from PowerRename code base.
-- Added a script for verifying the solution targets match the expected CPU architectures. Thanks [@snickler](https://github.com/snickler)!
-- Obsolete package Castle.Core was removed. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
-- Language typos were corrected across the PowerToys assets. Thanks [@pea-sys](https://github.com/pea-sys), [@eltociear](https://github.com/eltociear) and [@obairka](https://github.com/obairka)!
+- Fixed a build error that was restricting developers to switch between configuration without first cleaning local build files.
+- C++ exception catches were corrected to be caught by reference to avoind unneccessary copy operations. Thanks [@NN---](https://github.com/NN---)!
+- General C# code clean up, format fixing and removal of unused code analysis suppressions.
+- Removed unneccessary `muxc` prefix from XAML files. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
+- Re-enabled tests that depend on WebView2.
+- Windows 11 tier 1 context menu packages now contain the "Microsoft.PowerToys" prefix.
 
-#### What is being planned for v0.63
+#### What is being planned for v0.64
 
-For [v0.63][github-next-release-work], we'll work on below:
+For [v0.64][github-next-release-work], we'll work on below:
 
 - Environment Variables Editor PowerToy
 - GPO policies for PowerToys
+- Utility to list which processes are using a file
```

Comment:
We could call the community lead Hosts file app + backup for settings.

Reply:
I've pushed the suggested changes.